script/Ciel/Day 7/07_01_CIEL07_1.txt

```diff
@@ -376,7 +376,7 @@ Now that Vlov's dead, and the catacombs have collapsed, it's just an abandoned b
 [sha:c4a25ffd246fde561dd943c29578c84aa9b2c4bd]{
 -- Page 27, Offset 14871.
 -- 「……そっか。本当に、ぜんぶ終わったんだ」
-"Right... It's all over, isn't it."
+"Right... It's all over, isn't it?"
```

Comment:
revert

Reply:
I thought we were getting rid of all of these?